src/js/components/Select/SelectContainer.js

```diff
@@ -97,13 +97,6 @@ const SelectContainer = forwardRef(
         optionsNode.children[activeIndex].focus();
     }, [activeIndex]);
 
```

Comment:
I think we could remove search from this useEffect to solve the issue
```suggestion

// adjust activeIndex when options change
    useEffect(() => {
      if (activeIndex === -1 && optionIndexesInValue.length) {
        setActiveIndex(optionIndexesInValue[0]);
      }
    }, [activeIndex, optionIndexesInValue]);
    
```

Reply:
This change didn't resolve the issue.

Reply:
Oops yes sorry about that I must not have been typing fast enough for the bug to happen when I tested!

Reply:
Oops yes sorry about that I must not have been typing fast enough for the bug to happen when I tested!